Sources/ServiceModelEntities/String+nameConversions.swift

```diff
@@ -85,4 +88,12 @@ public extension String {
 
         return normalizedName
     }
+    
+    static func escapeReservedWords(name: String) -> String {
```

Comment:
This might be more convenient as a non-static function

```
    func swiftReservedWordEscaped() -> String {
        if reservedWords.contains(self) {
            return "`\(self)`"
        }

        return self
    }
```

Reply:
Good idea! Made the change